WalletWasabi.Fluent/ViewModels/Wallets/Buy/Workflows/ShopinBit/1_Initial/CountryStep.cs

```diff
@@ -33,7 +33,7 @@ public CountryStep(Conversation conversation, IReadOnlyList<Country> countries,
 	protected override IEnumerable<string> BotMessages(Conversation conversation)
 	{
 		// Assistant greeting, min order limit
-		yield return $"Hello, I am your chosen {GetAssistantName(conversation)}.\nFor now we only accept requests for legal goods or services of at least $1,000 USD.";
+		yield return $"Hello, I am your chosen {GetAssistantName(conversation)}.\nThe MINIMUM ORDER VALUE is $1,000 USD for now. We only accept requests for LEGAL goods or services.";
```

Comment:
This way it's smoother and sounds like less "robotic".

```suggestion
		yield return $"Hello, I am your chosen {GetAssistantName(conversation)}.\nFor now, the MINIMUM ORDER VALUE is $1,000 USD and we only accept requests for LEGAL goods or services.";
```